### PR TITLE
[DOC] Workflows Tutorial

### PR DESCRIPTION
**Pull request by skoudoro**
This PR is the first basic tutorial about DIPY Workflows. More to come on a new PR...

### CODE REVIEW

**Comment by arokem** [comment]
Small initial comment: I would reserve the word "pipeline" for future developments of more complex processing that automates the subsequent execution of several workflows together. In  other words, I would only use the word "workflow" here.

**Comment by skoudoro** [comment]
Ok, Thank you for feedback @arokem!  I Will update that now.

**Comment by Garyfallidis** [comment]
Looks good thanks.

File: doc/interfaces/basic_flow.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.. _basic_flow:
+
+====================================================
+Get Started with a basic DIPY Workflow command lines
+====================================================
+
+In a terminal, Let's get some data. This is multi-shell dataset which was
```
Comment (arokem) [inline doc/interfaces/basic_flow.rst]: "Let's" => "let's"

File: doc/interfaces/basic_flow.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Create stopping criteria::
+
+    dipy_fit_dti dwi.nii.gz dwi.bval dwi.bvec out_work/brain_mask.nii.gz --out_dir out_work/
+
+Create peaks::
+
+    dipy_fit_csd dwi.nii.gz dwi.bval dwi.bvec out_work/brain_mask.nii.gz --out_dir out_work/
+
+Create seeding mask (this creates seed_mask in new folder)::
```
Comment (arokem) [inline doc/interfaces/basic_flow.rst]: "in new folder" => "in a new folder"

File: doc/interfaces/basic_flow.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    dipy_mask out_work/fa.nii.gz 0.4 --out_dir out_work/ --out_mask seed_mask.nii.gz
+
+To avoid that we ommit the --out_dir option::
```
Comment (arokem) [inline doc/interfaces/basic_flow.rst]: "ommit" => "omit"

Comment (arokem) [reply doc/interfaces/basic_flow.rst]: I would add backticks around '--out_dir'

File: doc/documentation.rst [hunk 4]
```diff
@@ -12,6 +12,7 @@ Contents:
    mission
    installation
    examples_index
+   interfaces/index
```
Comment (arokem) [inline doc/documentation.rst]: "interfaces" is also not a great word here, in my opinion. Why not "workflows"?

**Comment by arokem** [comment]
BTW - where does #1990 stand? This would presumably need to be integrated there.

**Comment by skoudoro** [comment]
@Garyfallidis will update the typo and rephrase this basic tutorial on a new PR indeed. Thank you for all the corrections.

Concerning #1990, it will be easily integrated. I just have a small concern about the thumbnail that we can not generate. I do not think it is a good idea to run workflows script on real data to get a thumbnail (can be heavy a long computation). This part needs some thinking about the way we want to report workflow results.

**Pull request merged**